config/containerservice/config.go

```diff
@@ -49,6 +49,11 @@ func Configure(p *config.Provider) {
 		r.TerraformResource.Schema["kube_config"].Elem.(*schema.Resource).
 			Schema["password"].Sensitive = true
 
+		r.References["key_management_service.key_vault_key_id"] = config.Reference{
+			Type:      rconfig.VaultKeyReferencePath,
```

Comment:
Could you please use terraform name?

```suggestion
			TerraformName: "azurerm_key_vault_key",
```

As an example: https://github.com/upbound/provider-azure/blob/cbb65a75a29fb99d5edd0a02dfbbd2e9afc5fd09/config/network/config.go#L636